app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,3 @@
+class DashboardController < ApplicationController
+  def index; end
```

Comment:
Just because this might be used by less experienced people I'd leave a comment here

Reply:
Do you mean the comments that rails adds by default? or what kind of comment? I actually created this as a placeholder because I needed a root for the app after sign in/up

Reply:
I was thinking
```suggestion
  def index
    # just needed to render the action
  end
```

I've seen one-line methods confuse beginners who try to delete them since they "do nothing". But if this will go away or get variables it's fine